app/models/spotlight/custom_field.rb

```diff
@@ -50,12 +50,12 @@ def configured_to_display?
         end
     end
 
-    def solr_field
-      if field && field.starts_with?(solr_field_prefix)
+    def solr_field(document_model)
```

Comment:
Was there a problem pulling this out of the exhibit's `blacklight_configuration`?